packages/services/__tests__/content.ts

```diff
@@ -8,11 +8,24 @@ import {expected as latestExpected, response as latestResponse} from './fixtures
 import {expected as bookExpected, response as bookResponse} from './fixtures/bookmarks';
 import {expected as feedPubsExpected, response as feedPubsResponse} from './fixtures/feedPubs';
 
+import { post } from '../src/graphql/index';
+
 axios.defaults.adapter = httpAdapter;
 
 const baseURL = 'http://localhost:3000';
 const service = new ContentServiceImpl(baseURL, 5);
 
+function encode(val: string) {
+    return encodeURIComponent(val).
+        replace(/%40/gi, '@').
+        replace(/%3A/gi, ':').
+        replace(/%24/g, '$').
+        replace(/%2C/gi, ',').
+        replace(/%20/g, '+').
+        replace(/%5B/gi, '[').
+        replace(/%5D/gi, ']');
+}
```

Comment:
What about using `encodeURI` instead of `encodeURIComponent` with all the manual regex replacements?

Reply:
This is the code extracted from [axios](https://github.com/axios/axios/blob/master/lib/helpers/buildURL.js#L5-L14) in order to make the request work with `nock`, while writing tests. I'm not sure though why they decided to do the replacements... :sweat_smile:.

Reply:
Hmmm let's try `encodeURI`. I prefer to not keep such "magic" snippets if not necessary and if indeed necessary let's add a comment. Please check and let me know

Reply:
Hi! Sorry for the delay! I just checked and it works fine with `encodeURI`. 
I'll add some comments and then I'll update the PR.

Reply:
Sounds legit. Let's keep it this way just add a comment explaining this :)